spotify/models/podcast.py

```diff
@@ -0,0 +1,85 @@
+from functools import partial
+from itertools import starmap
+from typing import List, Optional, Union
+
+from . import AsyncIterable, URIBase, Image
+from ..http import HTTPClient
+
+
+class Episode(URIBase):
+    """A Spotify Episode Object."""
+
+    def __init__(self, client, data):
+
+        self.__client = client
+
+        self.preview_url = data.pop("audio_preview_url", None)
+        self.description = data.pop("description", None)
+        self.duration = data.pop("duration_ms", None)
+        self.explicit = data.pop("explicit", None)
+        self.external_urls = data.pop("external_urls").get("spotify", None)
+        self.href = data.pop("href", None)
+        self.id = data.pop("id", None)
+        self.externally_hosted = data.pop("is_externally_hosted", None)
+        self.playable = data.pop("is_playable", None)
+        self.language = data.pop("language", None)
+        self.name = data.pop("name", None)
+        self.release_date = data.pop("release_date", None)
+        self.release_date_presicion = data.pop("release_date_precision", None)
+        self.type = data.pop("episode", None)
+        self.uri = data.pop("uri", None)
+
+        show_ = data.pop("show", None)
+        self.show = show = show_ and Show(client, show_)
+
+        if "images" in data:
+            self.images = list(starmap(Image, data.pop("images")))
+        else:
+            self.images = show.images.copy() if show is not None else []
+
+    def __repr__(self) -> str:
+        return f"<spotify.Episode: {self.name!r}>"
+
+    def __str__(self) -> str:
+        return self.id
+
+
+class Show(URIBase, AsyncIterable):
```

Comment:
I'm not sure how should i proceed after, can't wrap the abstraction in my head, Should we create a `Podcast` class?

As far as i understand from the API documentation, Podcast's are actually a **Show**.

Reply:
I'd ditch the Podcast class and add (if possible) add a method to Show, something like `def is_podcast(self) -> bool` or something